backend/src/test/java/org/cryptomator/hub/api/VaultResourceTest.java

```diff
@@ -585,7 +590,8 @@ public void setup() throws SQLException {
 							('user91'),
 							('user92'),
 							('user93'),
-							('user94');
+							('user94'),
+							('userAA');
```

Comment:
Do we really need a different user?

If yes: do we really need a different naming scheme for the IDs?

Reply:
Yes, because this user is part of an archived vault. I deliberately changed the naming scheme to indcate what the user is used for.

Reply:
But i guess i can adapt more the naming scheme. Will change it.

Reply:
I would really like to keep it consistent, keeping different number ranges for different test cases. With the "low numbers", defined in `V9999__Test_Data.sql` being "reusable", so they should only be access for read-only test cases. Higher numbers being test-specific, volatile and should not overlap.